core/src/main/java/com/sap/ai/sdk/core/Core.java

```diff
@@ -282,10 +282,15 @@ private static boolean isDeploymentOfModel(
     if (resources == null) {
       return false;
     }
-    if (!resources.getCustomFieldNames().contains("backend_details")) {
-      return false;
+    Object detailsObject = resources.getBackendDetails();
+    // workaround for https://jira.tools.sap/browse/AIWDF-2124
```

Comment:
Add TODO to the comment. Also I was told to remove internal links on this repo.

Reply:
adding a todo makes the build fail..